cfn-resources/search-deployment/.rpdk-config

```diff
@@ -0,0 +1,22 @@
+{
+    "artifact_type": "RESOURCE",
+    "typeName": "MongoDB::Atlas::SearchDeployment",
+    "language": "go",
+    "runtime": "provided.al2",
+    "entrypoint": "bootstrap",
+    "testEntrypoint": "bootstrap",
+    "settings": {
+        "version": false,
+        "subparser_name": null,
+        "verbose": 0,
+        "force": false,
+        "type_name": "MongoDB::Atlas::SearchDeployment",
+        "artifact_type": "r",
+        "endpoint_url": null,
+        "region": null,
+        "target_schemas": [],
+        "profile": null,
+        "import_path": "github.com/mongodb/mongodbatlas-cloudformation-resources/search-deployment",
```

Comment:
this path doesn't exist, not sure what it's used for

Reply:
did some investigation here and it appears to be the correct way (also checked that all resources follow this convention). This value is provided during the initial generation process (https://github.com/mongodb/mongodbatlas-cloudformation-resources/tree/master/cfn-resources/autogen#step-3) and the format it correct, it is used for constructing the import path in main.go file of the created resource: https://github.com/mongodb/mongodbatlas-cloudformation-resources/blob/master/cfn-resources/project/cmd/main.go#L11